source/css/_common/components/post/post-expand.styl

```diff
@@ -49,7 +49,7 @@
     margin-bottom: 20px;
   }
 
-  .video-container {
```

Comment:
It should be `.video-container`, see
https://github.com/hexojs/hexo/blob/0b26940f7e0a35f6937b944012468bbf0d844010/lib/plugins/tag/vimeo.js
https://github.com/hexojs/hexo/blob/0b26940f7e0a35f6937b944012468bbf0d844010/lib/plugins/tag/youtube.js